README.md

```diff
@@ -64,6 +78,17 @@ Or you can also define a `jest-junit` key in your `package.json`.  All are **str
 }
 ```
 
+Or you can define your options in your reporter configuration.
+
+```
+// jest.config.js
+{
+	reporters: [
+    	"jest-junit/reporter", { suiteName: "jest tests" }
```

Comment:
to pass config, you have to make it an array (otherwise your config would be just another reporter).

```js
// jest.config.js
{
	reporters: [
    	["jest-junit/reporter", { suiteName: "jest tests" }]
    ]
}
```

Reply:
Awesome thanks for catching this!